### PR TITLE
[zh] sync content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

### PR DESCRIPTION
**Pull request by yuandongx**
Signed-off-by: yuandongx <yuandongx@126.com>

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 1]
```diff
@@ -565,6 +566,26 @@ Kubernetes 的默认约束。
 插件 `PodTopologySpread` 不会为未设置分布约束中所给拓扑键的节点评分。
 {{< /note >}}
 
+<!--
+If you don't want to use the default Pod spreading constraints for your cluster,
+you can disable those defaults by setting `defaultingType` to `List` and leaving
+empty `defaultConstraints` in the `PodTopologySpread` plugin configuration:
+-->
+如果你不想为群集使用默认的 Pod 分布约束，你可以通过设置 `defaultingType` 参数为 `List` 和
```
Comment (howieyuen) [inline content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:574]: 集群

**Review by howieyuen — CHANGES_REQUESTED** [review]
please modify the commit message(now it's only a word 'sync') and PR title

**Comment by howieyuen** [comment]
/retitle [zh] sync content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

**Comment by howieyuen** [comment]
> /assign howieyuen

please update the commit message, now valuable word is only "sync"

**Comment by yuandongx** [comment]
> > /assign howieyuen
> 
> please update the commit message, now valuable word is only one : "sync"

OK, done.

**Pull request merged**